nixos/modules/programs/rust-motd.nix

```diff
@@ -50,9 +103,10 @@ in {
       path = with pkgs; [ bash ];
       documentation = [ "https://github.com/rust-motd/rust-motd/blob/v${pkgs.rust-motd.version}/README.md" ];
       description = "motd generator";
+      wantedBy = [ "multi-user.target" ];
```

Comment:
I suppose this isn't related to the rest of the change?

Reply:
Yes, correct. This was more of a "why didn't I do it already that way"-moment and that's also the reason why this is in its own commit. If everybody agrees on that, I'd say we'll merge it right in, otherwise I'm happy to remove it from this branch and open up another PR for it.

Reply:
I think it's fine having it in this PR. As you say, it lives in its own commit anyways.